test/distributed/cases/function/func_llm_extract_file.sql

```diff
@@ -0,0 +1,2 @@
+select llm_extract_text(cast('file://$resources/llm_test/extract_text/MODocs1.pdf?offset=0&size=4' as datalink), cast('file://$resources/llm_test/extract_text/MODocs1.txt' as datalink), "pdf");
```

Comment:
Why do we need offset and size here?

Reply:
I included them here to demonstrate that this format of datalink is recognized by the function. 
Offset and size are not utilized in the llm_extract_text function and will not affect the outcome.
Shall I delete them?

Reply:
Yeah, just remove those.